src/ui/storybook/iconsExports.ts

```diff
@@ -261,6 +263,8 @@ export const TertiaryAndSmallerIcons = {
   Parameters,
   PhoneFilled,
   PlainArrowNext,
+  CheckboxMark: CheckboxMarkChecked,
```

Comment:
pourquoi ne pas le nommer comme ce qu'on a dans le code ?

```suggestion
  CheckboxMarkChecked,
```